controllers/tests/controller/selfnoderemediation_controller_test.go

```diff
@@ -52,7 +52,6 @@ var _ = Describe("SNR Controller", func() {
 		}
 
 		updateIsRebootCapable(nodeRebootCapable)
-		createSNR(snr, remediationStrategy)
```

Comment:
I think that a more elegant way to handle it would be to keep this method here.
We could set a flag in "BeforeEach" method for the new test that this method will check and setup the SNR accordingly

Reply:
It is not exactly a new test, but the same test run twice with different SNR. This is possible thanks to `DescribeTable`, but it is evaluated _after_ the `BeforeEach`, so its `Entries` cannot drive the flag in `BeforeEach`.

The alternative would be to move the content of "Automatic strategy - OutOfServiceTaint selected" into a function and run it first in a Context with the BeforeEach setting the flag for a "normal" SNR and then in another context whose BeforeEach sets the flag for the SNR with support for multiple-kind-of-same-type.

I lean towards the current version, but not strongly :)

Reply:
we discussed further in chat, and now I got what can be improved :)
I'll implement a flag that will skip the creation of SNR in the `JustBeforeEach` node only for the test using `DescribeTable`, so that the other tests won't be affected anymore.